pkg/providers/securitygroup/suite_test.go

```diff
@@ -113,80 +113,78 @@ var _ = AfterEach(func() {
 var _ = Describe("Security Group Provider", func() {
 	It("should default to the clusters security groups", func() {
 		ExpectApplied(ctx, env.Client, provisioner, nodeTemplate)
-		resolvedSecurityGroups, err := awsEnv.SecurityGroupProvider.List(ctx, nodeTemplate)
+		securityGroups, err := awsEnv.SecurityGroupProvider.List(ctx, nodeTemplate)
 		Expect(err).To(BeNil())
-		Expect(len(resolvedSecurityGroups)).To(Equal(3))
-		Expect(resolvedSecurityGroups).To(ConsistOf(
-			"sg-test1",
-			"sg-test2",
-			"sg-test3",
-		))
+		Expect(len(securityGroups)).To(Equal(3))
+		Expect(aws.StringValue(securityGroups[0].GroupId)).To(Equal("sg-test1"))
```

Comment:
nit: Rather than just ensuring each value, can you ensure the entire struct is equal?

Reply:
I will make a separate PR for the subnets status testing to make similar changes.